packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -34,16 +35,26 @@ const getBitcoinFeeInfo = (info: FeeInfo, feeRate: string) => {
 
 const getEthereumFeeInfo = (info: FeeInfo, gasPrice: string) => {
     const current = new BigNumber(gasPrice);
-    const minFee = current.plus(1);
+    const minFeeFromNetwork = new BigNumber(
+        fromWei(info.levels[0].feePerUnit, 'gwei'),
+    ).integerValue(BigNumber.ROUND_FLOOR);
+
+    const getMinFee = () => {
+        if (minFeeFromNetwork.toNumber() <= current.toNumber()) {
```

Comment:
```suggestion
        if (minFeeFromNetwork.lte(current)) {
```

I believe this is better and it should be equivalent